code/modules/bs_snare/snare.dm

```diff
@@ -5,10 +5,12 @@
 	icon_state = "teleporter"
 	w_class = ITEM_SIZE_SMALL
 	origin_tech = list(TECH_BLUESPACE = 6)
-
 	//var/locking_access = access_rd
 	var/silk_id = "" //using by snare controller to teleport user to controller's mark
 
+/obj/item/clothing/accessory/bs_silk/New()
+	. = ..()
+	matter = list(MATERIAL_STEEL = 8, MATERIAL_GLASS = 5, MATERIAL_SILVER = 10)
```

Comment:
Why do it in New instead of just overriding list in the type definition? Doesn't seem to be using anything dynamic, should Just Work.

Reply:
Snare mustn't have biomatter in it matter